core-bundle/src/Image/Studio/FigureBuilder.php

```diff
@@ -351,6 +351,8 @@ public function setLocale(?string $locale): self
      *
      * Set the value to null to remove it. If you want to explicitly remove an
      * auto-generated value from the results, set the $forceRemove flag to true.
+     *
+     * @param string|int|bool $forceRemove
```

Comment:
```suggestion
     * @param bool $forceRemove
```
why not `bool` only?

Reply:
Changed in 2e46f7a8ef9084c1ecd417917cf7717745d76c99.